packages/client/src/components/LocationPicker.tsx

```diff
@@ -116,6 +118,7 @@ function LocationPickerComponent(props: LocationPickerProps) {
   const offlineSearchableOffices = generateLocations(offlineOffices, intl)
 
   const searchableLocations = [
+    ...additionalLocations,
```

Comment:
I think this additionalLocations have to be added in the PerformanceHome searchableLocations.